medicines/api/Makefile

```diff
@@ -57,7 +57,7 @@ docker-run: ## Run Docker image
 
 .PHONY: docker-pull
 docker-pull: ## Pull Docker image
-	docker pull $(image)
+	docker pull $(image):$(tag)
```

Comment:
couldn't you just do  `make pull image="imagename:tagname"` without needing the second var?

Reply:
I think it makes sense to list out the tag separately as that's what is done elsewhere in this makefile so it makes the argument values consistent then between commands